### ISSUE TITLE
install function in ProfileModuleInstaller, request stack, and command-line 

### ISSUE THREAD

**Issue by isolic811** [zikula-modules/Profile#105]
Hi,

I'm using the Profile module in my Zikula.

Everything works fine when I install module from the Extension Module and click Install button that redirects to installation route.

When I go to install module from a CLI context (I created new command that in one step triggers install function of module), things go bad:

![image](https://user-images.githubusercontent.com/17670016/64244076-b717f780-cf08-11e9-8e74-b2b46ae38665.png)

This error refers to creating default data for the Profile module in its install function. 

`      
        $locale = $this->container->get('request_stack')->getMasterRequest()->getLocale();
`
`
        $this->defaultdata($locale);
`

If we install it from Extensions module, everything is ok because there is request and you can call function getLocale(), but there is no request when I call this function with Command.

Is there a way to a working CLI installation of the module without hardcoding $locale variable?

**Comment by Guite** [comment]
I think hardcoding is the best option without huge efforts.

Can you please try this:
```
$requestStack = $this->container->get('request_stack');
$request = null !== $requestStack ? $requestStack->getMasterReqest() : null;
$locale = null !== $request ? $requestStack->getLocale() : 'en';
```

Note this is actually only used for inserting default data, so the hardcoded value does not really hurt.

**Comment by isolic811** [comment]
Works fine, thanks! :)

**Comment by Guite** [comment]
done

**Comment by isolic811** [comment]
You have typo in added line: 
$request = null !== $requestStack ? $requestStack->getMasterReqest() : null;

Should be:

$request = null !== $requestStack ? $requestStack->getMasterRequest() : null;

**Comment by Guite** [comment]
Well spotted, thank you. Fixed in  9c473d2

**Comment by craigh** [comment]
@isolic811 please consider submitting your CLI extension installer to the core via pull request.
thanks!
